test/unit/microcosm/patch.test.js

```diff
@@ -98,6 +98,72 @@ describe('Microcosm::patch', function () {
       // the new state produced by a child.
       expect(child.state.count).toEqual(4)
     })
+
+    it('does not strip away parent inherited state', function () {
+      const parent = new Microcosm()
+      const child = parent.fork()
+
+      parent.addDomain('top', {
+        getInitialState() {
+          return false
+        }
+      })
+
+      child.addDomain('bottom', {
+        getInitialState() {
+          return false
+        }
+      })
+
+      child.patch({ bottom: true })
```

Comment:
question about `patch`: in this example, if the domain state is an object, what's the expected output?

```js
child.addDomain('example', {
  getInitialState() { return { id: 1, name: 'Billy' } }
}

child.patch({ example: { name: 'Ben' } })
```

I think I would expect `example` to be `{ id: 1, name: 'Ben' }` but wanted to check and see if there's a test for that already.